tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite.py

```diff
@@ -0,0 +1,115 @@
+import pathlib
+import great_expectations as gx
+import tempfile
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+
+from great_expectations.core.expectation_suite import ExpectationSuite
+
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_context">
+import great_expectations as gx
+
+context = gx.get_context()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite create_validator">
+validator = context.sources.pandas_default.read_csv(
+    "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+)
+# </snippet>
+
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite add_2_expectations">
+validator.expect_column_values_to_not_be_null("pickup_datetime")
+validator.expect_column_values_to_be_between("passenger_count", auto=True)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_suite">
+my_suite = validator.get_expectation_suite()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite show_suite">
+my_suite.show_expectations_by_expectation_type()
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite example_dict_1">
+
+{
+    "expect_column_values_to_be_between": {
+        "auto": True,
+        "column": "passenger_count",
+        "domain": "column",
+        "max_value": 6,
+        "min_value": 1,
+        "mostly": 1.0,
+        "strict_max": False,
+        "strict_min": False,
+    }
+}
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite example_configuration_1">
+config = ExpectationConfiguration(
+    expectation_type="expect_column_values_to_be_between",
+    kwargs={
+        "auto": True,
+        "column": "passenger_count",
+        "domain": "column",
+        "max_value": 6,
+        "min_value": 1,
+        "mostly": 1.0,
+        "strict_max": False,
+        "strict_min": False,
+    },
+)
+# </snippet>
+
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite updated_configuration">
+updated_config = ExpectationConfiguration(
+    expectation_type="expect_column_values_to_be_between",
+    kwargs={
+        "auto": True,
+        "column": "passenger_count",
+        "domain": "column",
+        "max_value": 4,
+        #'max_value': 6,
+        "min_value": 1,
+        "mostly": 1.0,
+        "strict_max": False,
+        "strict_min": False,
+    },
+)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite add_configuration">
+my_suite.add_expectation(updated_config)
```

Comment:
How about an assert here on the contents of my_suite?

Reply:
done!